modules/backend/widgets/Filter.php

```diff
@@ -561,6 +562,12 @@ public function addScopes(array $scopes)
         foreach ($scopes as $name => $config) {
             $scopeObj = $this->makeFilterScope($name, $config);
 
+            // Check if user has permission to show this filter
+            $permission = array_get($config, 'permission');
+            if ($permission && !BackendAuth::getUser()->hasAccess($permission)) {
```

Comment:
```suggestion
            if ($permission && !BackendAuth::getUser()->hasAnyAccess($permission)) {
```
Should be `hasAnyAccess()` instead of `hasAccess()`, see https://github.com/octobercms/october/blob/master/modules/backend/classes/NavigationManager.php#L520

Reply:
Will need to be changed for all the permission checks (lists, forms, filters)

Reply:
Can you then write it in single line if its single permission or then its needed to write it for everything as array?

Reply:
@Samuell1 permissions can still be defined as a string (`permissions: backend.manage_users` for example) as hasAnyAccess just calls hasAccess($permission, $all = false) which calls hasPermission() which takes an array or a string as input: https://github.com/octobercms/library/blob/master/src/Auth/Models/User.php#L460

Reply:
@Samuell1 the method called needs to be changed to `hasAnyAccess()` not just `hasAccess()`